lib/ddtrace/contrib/resque/resque_job.rb

```diff
@@ -9,11 +9,11 @@ module Contrib
     module Resque
       # Uses Resque job hooks to create traces
       module ResqueJob
-        def around_perform(*_)
+        def around_perform(args = {})
```

Comment:
Probably better to leave this as `*args`; future releases of Resque could change the number of parameters provided to this function, which would cause `ArgumentError`s.

Instead I think you can check `args.first` to see if its the right kind of argument you want to use.